sql/analyzer/resolve_column_defaults.go

```diff
@@ -0,0 +1,480 @@
+	"st_geohash": {},
+	"st_geomcollfromtext": {},
+	"st_geomcollfromtxt": {},
+	"st_geomcollfromwkb": {},
+	"st_geometrycollectionfromtext": {},
+	"st_geometrycollectionfromwkb": {},
+	"st_geometryn": {},
+	"st_geometrytype": {},
+	"st_geomfromgeojson": {},
+	"st_geomfromtext": {},
+	"st_geometryfromtext": {},
+	"st_geomfromwkb": {},
+	"st_geometryfromwkb": {},
+	"st_interiorringn": {},
+	"st_intersection": {},
+	"st_intersects": {},
+	"st_isclosed": {},
+	"st_isempty": {},
+	"st_issimple": {},
+	"st_isvalid": {},
+	"st_latfromgeohash": {},
+	"st_latitude": {},
+	"st_length": {},
+	"st_linefromtext": {},
+	"st_linestringfromtext": {},
+	"st_linefromwkb": {},
+	"st_linestringfromwkb": {},
+	"st_longfromgeohash": {},
+	"st_longitude": {},
+	"st_makeenvelope": {},
+	"st_mlinefromtext": {},
+	"st_multilinestringfromtext": {},
+	"st_mlinefromwkb": {},
+	"st_multilinestringfromwkb": {},
+	"st_mpointfromtext": {},
+	"st_multipointfromtext": {},
+	"st_mpointfromwkb": {},
+	"st_multipointfromwkb": {},
+	"st_mpolyfromtext": {},
+	"st_multipolygonfromtext": {},
+	"st_mpolyfromwkb": {},
+	"st_multipolygonfromwkb": {},
+	"st_numgeometries": {},
+	"st_numinteriorring": {},
+	"st_numinteriorrings": {},
+	"st_numpoints": {},
+	"st_overlaps": {},
+	"st_pointfromgeohash": {},
+	"st_pointfromtext": {},
+	"st_pointfromwkb": {},
+	"st_pointn": {},
+	"st_polyfromtext": {},
+	"st_polygonfromtext": {},
+	"st_polyfromwkb": {},
+	"st_polygonfromwkb": {},
+	"st_simplify": {},
+	"st_srid": {},
+	"st_startpoint": {},
+	"st_swapxy": {},
+	"st_symdifference": {},
+	"st_touches": {},
+	"st_transform": {},
+	"st_union": {},
+	"st_validate": {},
+	"st_within": {},
+	"st_x": {},
+	"st_y": {},
+	"statement_digest": {},
+	"statement_digest_text": {},
+	"std": {},
+	"stddev": {},
+	"stddev_pop": {},
+	"stddev_samp": {},
+	"str_to_date": {},
+	"strcmp": {},
+	"subdate": {},
+	"substr": {},
+	"substring": {},
+	"substring_index": {},
+	"subtime": {},
+	"sum": {},
+	"sysdate": {},
+	"system_user": {},
+	"tan": {},
+	"time": {},
+	"time_format": {},
+	"time_to_sec": {},
+	"timediff": {},
+	"timestamp": {},
+	"timestampadd": {},
+	"timestampdiff": {},
+	"to_base64": {},
+	"to_days": {},
+	"to_seconds": {},
+	"trim": {},
+	"truncate": {},
+	"ucase": {},
+	"uncompress": {},
+	"uncompressed_length": {},
+	"unhex": {},
+	"unix_timestamp": {},
+	"updatexml": {},
+	"upper": {},
+	"user": {},
+	"utc_date": {},
+	"utc_time": {},
+	"utc_timestamp": {},
+	"uuid": {},
+	"uuid_short": {},
+	"uuid_to_bin": {},
+	"validate_password_strength": {},
+	"values": {},
+	"var_pop": {},
+	"var_samp": {},
+	"variance": {},
+	"version": {},
+	"wait_for_executed_gtid_set": {},
+	"wait_until_sql_thread_after_gtids": {},
+	"week": {},
+	"weekday": {},
+	"weekofyear": {},
+	"weight_string": {},
+	"year": {},
+	"yearweek": {},
+}
+
+func resolveColumnDefaults(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope) (sql.Node, error) {
+	span, _ := ctx.Span("resolveColumnDefaults")
+	defer span.Finish()
```

Comment:
n.Resolved() check here, since you are calling node.Schema(). The latter will panic if any expressions aren't resolved.